src/arm/deoptimizer-arm.cc

```diff
@@ -216,6 +216,8 @@ void Deoptimizer::TableEntryGenerator::Generate() {
   // Copy VFP registers to
   // double_registers_[DoubleRegister::kMaxNumAllocatableRegisters]
   int double_regs_offset = FrameDescription::simd128_registers_offset();
+  const RegisterConfiguration* config =
```

Comment:
It's not a blocker though, so I'll merge this to continue with the rebase. Thanks.